docs/installing-upgrading.md

```diff
@@ -48,23 +48,31 @@ Some pointers for common tasks:
 ### Option 1: Run REMS from dockerhub
 
     docker-compose up -d db
-    docker-compose run --rm -e COMMANDS="migrate test-data" app
+    docker-compose run --rm -e CMD="migrate;test-data" app
     docker-compose up -d app
 
 ### Option 1.1: Use config file simple-config.edn instead of environment variables
 
     docker-compose -f docker-compose-config.yml up -d db
-    docker-compose -f docker-compose-config.yml run --rm -e COMMANDS="migrate test-data" app
+    docker-compose -f docker-compose-config.yml run --rm -e CMD="migrate;test-data" app
     docker-compose -f docker-compose-config.yml up -d app
 
 ### Option 2: Build REMS image locally
 
     lein uberjar
     docker-compose -f docker-compose-build.yml build
     docker-compose -f docker-compose-build.yml up -d db
-    docker-compose -f docker-compose-build.yml run --rm -e COMMANDS="migrate test-data" app
+    docker-compose -f docker-compose-build.yml run --rm -e CMD="migrate;test-data" app
     docker-compose -f docker-compose-build.yml up -d app
 
+### CMD environment variable may be populated with any rems commands
+
+Command `run` as well as empty `CMD` (and `COMMANDS`) parameter starts rems. Example `CMD` varibale values:
```

Comment:
"varibale"

I think the structure of the sentence could be a bit clearer, perhaps something like:

> You can use the CMD environment variable to specify a sequence of REMS administration commands to run. An empty CMD or the command `run` starts the REMS server.

Reply:
Updated